Error on Scrape Case Data

``` 
Run npm run case

> case
> node ./case.js --unhandled-rejections=strict

simple .rpsa_detail > div > #mapAll
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at /home/runner/work/corcc/corcc/GET.js:85:28
    at Array.map (<anonymous>)
    at mohwJson (/home/runner/work/corcc/corcc/GET.js:83:50)
    at /home/runner/work/corcc/corcc/GET.js:123:24
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
Error: Process completed with exit code 255.
```

--- a/.github/workflows/scrp.yml
+++ b/.github/workflows/scrp.yml
@@ -59,33 +59,34 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} 
         TASK_NAME: '⬆️💉'
         TIMEZONE: 'Asia/Tokyo'
-  case:
-    name: ⬆️🦠
-    needs: [node-dep]
-    runs-on: ubuntu-latest
-    steps:
-    - name: Setup Node
-      uses: actions/setup-node@v2
-      with:
-        node-version: 'lts/*'
-    - name: Checkout 🛎️
-      uses: actions/checkout@master
-    - name: Cache node_modules 🔧⚙️🟢
-      uses: actions/cache@v2
-      id: cache-modules
-      with:
-        path: node_modules
-        key: ${{ runner.os }}-modules-${{ hashFiles('package-lock.json')}}
-    - name: ⬆️🦠
-      run: npm run case 
-    - uses: corcc/publish@master
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} 
-        TASK_NAME: '⬆️🦠'
-        TIMEZONE: 'Asia/Tokyo'
+# discontinued
+#   case:
+#     name: ⬆️🦠
+#     needs: [node-dep]
+#     runs-on: ubuntu-latest
+#     steps:
+#     - name: Setup Node
+#       uses: actions/setup-node@v2
+#       with:
+#         node-version: 'lts/*'
+#     - name: Checkout 🛎️
+#       uses: actions/checkout@master
+#     - name: Cache node_modules 🔧⚙️🟢
+#       uses: actions/cache@v2
+#       id: cache-modules
+#       with:
+#         path: node_modules
+#         key: ${{ runner.os }}-modules-${{ hashFiles('package-lock.json')}}
+#     - name: ⬆️🦠
+#       run: npm run case 
+#     - uses: corcc/publish@master
+#       env:
+#         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} 
+#         TASK_NAME: '⬆️🦠'
+#         TIMEZONE: 'Asia/Tokyo'
   numberize:
     name: ⬆(0️⃣1️⃣2️⃣3️⃣4️⃣5️⃣6️⃣7️⃣8️⃣9️⃣)
-    needs: [node-dep,vaccination,case]
+    needs: [node-dep,vaccination]
     runs-on: ubuntu-latest
     steps:
     - name: Setup Node
@@ -111,7 +112,7 @@ jobs:
         TIMEZONE: 'Asia/Tokyo'
   vc:
     name: ⬆️💉🦠
-    needs: [node-dep,vaccination,case,numberize]
+    needs: [node-dep,vaccination,numberize]
     runs-on: ubuntu-latest
     steps:
     - name: Setup Node

--- a/README.md
+++ b/README.md
@@ -2,7 +2,7 @@
 ## FROM \[
 > [ COVID-19 Vaccination ](https://nip.kdca.go.kr/irgd/cov19stats.do) (XML)
 > 
-> [ Cases in Korea by City/Province (**NOT-SECURED**) ](http://ncov.mohw.go.kr/en/bdBoardList.do?brdGubun=162) (HTML)
+> ~~~[ Cases in Korea by City/Province (**NOT-SECURED**) ](http://ncov.mohw.go.kr/en/bdBoardList.do?brdGubun=162) (HTML)~~~ (DISCONTINUED on 2021-11-17 => REASON : NO DATA )[Archive](https://web.archive.org/web/20211116205419/http://ncov.mohw.go.kr/en/bdBoardList.do?brdId=16&brdGubun=162&dataGubun=&ncvContSeq=&contSeq=&board_id=&gubun=)
 > 
 ## \]
 to json() thngs

--- a/package.json
+++ b/package.json
@@ -1,6 +1,5 @@
 {
   "scripts": {
-    "vaccination": "node ./vaccination.js --unhandled-rejections=strict",
     "case": "node ./case.js --unhandled-rejections=strict",
     "numberize": "node ./numberize.js --unhandled-rejections=strict",
     "vc": "node ./vc.js --unhandled-rejections=strict"

